src/py/flwr/common/constant.py

```diff
@@ -48,6 +48,7 @@
 )
 FLEET_API_REST_DEFAULT_ADDRESS = "0.0.0.0:9095"
 EXEC_API_DEFAULT_ADDRESS = "0.0.0.0:9093"
+SIMULATIONIO_API_DEFAULT_ADDRESS = "0.0.0.0:9094"
```

Comment:
Proposal to change the default to avoid overlap with `CLIENTAPPIO_API_DEFAULT_ADDRESS`:
```suggestion
SIMULATIONIO_API_DEFAULT_ADDRESS = "0.0.0.0:9096"
```